src/features/MoviePage/Content/MainInfo/index.js

```diff
@@ -29,8 +29,6 @@ import noMovieImage from "../../images/no-movie-image.svg";
 const MainInfo = () => {
   const movieDetailsData = useSelector(selectMovieDetailsData);
   const movieDetailsStatus = useSelector(selectMovieDetailsStatus);
-
-  const posterBaseUrl = "http://image.tmdb.org/t/p/w342";
```

Comment:
I liked this decision more, but as you wish😉

Reply:
I assumed that it's used just ones in this component so it's not necessary to do separate variable